### PR TITLE
aodh: add aodh proposal

### PR DESCRIPTION
**Pull request by sidthe**
need to enable aodh for the new barclamp

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -1077,6 +1077,8 @@ Optional
         It's not deployed by default. Set to 1 to deploy murano.
     want_trove=0 (default=1)
         Deployed by default. Set to 0 to prevent trove from being deployed.
+    want_aodh=1 (default=0)
```
Comment (Itxaka) [inline scripts/mkcloud]: default=1 ?

**Comment by sidthe** [comment]
[(5) Split aodh out of ceilometer barclamp](https://trello.com/c/ZtGoKQwG/403-5-split-aodh-out-of-ceilometer-barclamp)

File: scripts/mkcloud [hunk 2]
```diff
@@ -1077,6 +1077,8 @@ Optional
         It's not deployed by default. Set to 1 to deploy murano.
     want_trove=0 (default=1)
         Deployed by default. Set to 0 to prevent trove from being deployed.
+    want_aodh=0 (default=0)
```
Comment (mjura) [inline scripts/mkcloud]: Should be `want_aodh=1 (default=0)`

File: scripts/mkcloud [hunk 3]
```diff
@@ -1077,6 +1077,8 @@ Optional
...
+        Deployed by default. Set to 0 to prevent aodh from being deployed.
```
Comment (mjura) [inline scripts/mkcloud]: It is not deployed by default

File: scripts/mkcloud (hunk 2 above)
Comment (sidthe) [inline scripts/mkcloud]: let's keep it as 0 for the time being so I can only apply it for my jobs testing. Once all works, will change to 1

Comment (mjura) [inline scripts/mkcloud]: I still have doubts about if we should add new option there is already mechanism to skip proposals byt defining `want_name_proposal` https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crowbarsetup.sh#L3010.
For me add new option is fine, but some other my PR's were rejected with new `want_` options

**Pull request merged**